clients/src/main/java/org/apache/kafka/clients/NetworkClient.java

```diff
@@ -839,20 +833,22 @@ private void handleCompletedReceives(List<ClientResponse> responses, long now) {
             InFlightRequest req = inFlightRequests.completeNext(source);
             Struct responseStruct = parseStructMaybeUpdateThrottleTimeMetrics(receive.payload(), req.header,
                 throttleTimeSensor, now);
-            if (log.isTraceEnabled()) {
```

Comment:
Letting some requests be debug level, but making the responses is trace often means we are sometimes left with only half of the picture.